llvm/test/Transforms/LoopVectorize/no-fold-tail-by-masking-iv-external-uses.ll

```diff
@@ -0,0 +1,85 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -passes=loop-vectorize -S | FileCheck %s
```

Comment:
As this is added as a target-independent test, it probably needs something like `-force-vector-width=4 -force-vector-interleave=1` to make sure the vectorizer tries to vectorize independent of the cost-model.